src/dispatch/plugins/dispatch_slack/plugin.py

```diff
@@ -71,6 +74,34 @@ def create(self, name: str):
         client = create_slack_client(self.configuration)
         return create_conversation(client, name, self.configuration.private_channels)
 
+    def create_threaded(self, case: Case, conversation_id: str):
+        """Creates a new threaded conversation."""
+        client = create_slack_client(self.configuration)
+        blocks = create_case_message(case=case, channel_id=conversation_id)
+        response = send_message(client=client, conversation_id=conversation_id, blocks=blocks)
+        send_message(
+            client=client,
+            conversation_id=conversation_id,
+            text="All real-time case collaboration should be captured in this thread.",
+            ts=response["timestamp"],
+        )
+        if case.signal_instances:
+            messages = create_signal_messages(case=case)
+            for m in messages:
```

Comment:
I imagine there could be a lot of these and the thread could get a bit cluttered. Should we try to summarize and link out to the case view on the front-end?

Reply:
Yes, I don't know about the balance here. I figured we'd know more once we run through some of these. It might also depend on the signal itself (if it's likely to have a lot of related signals).

Reply:
Yeah, probably not worth solving up front.